nfc/NfcCx/Cx/NfcCxSC.cpp

```diff
@@ -334,6 +334,8 @@ Return Value:
                                          &GUID_DEVINTERFACE_SMARTCARD_READER,
                                          &nfcScReaderReference,
                                          TRUE);
+
+        NfcCxEmbeddedSEInterfaceStart(fdoContext->SCInterface);
```

Comment:
this call will create and enable the eSE pcsc interface. A comment will be added in the next commit